tools/verify_stable_archives.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from urllib.parse import urlparse
+
+from registry import RegistryClient
+
+def verify_stable_archive(url):
+  parsed = urlparse(url)
+  path_parts = parsed.path.split("/")
```

Comment:
On that front, I wasn't sure how opinionated the central registry is. As we are looking to have stable URLs, are we also assuming that all source URLs are hosted on GitHub, or are we just trying to verify that of all the source URLs that are hosted on GitHub we are using the known stable form?

I adjusted it now to only check `https://github.com` URLs, and otherwise assume that the provided URL is stable.